app/views/application/_browsing_index.html.erb

```diff
@@ -5,9 +5,9 @@
       <div class="col-md-3 p-4">
         <%= link_to image_tag(obit.image_url), "/obituaries/#{obit.id}" %>
       </div>
-      <div class="col-md-9 p-4">
+      <div class="col-md-9 p-4"%>
```

Comment:
Does the percent do something here?

Reply:
this might just be a typo let me fix and repush